errors/grpc.go

```diff
@@ -0,0 +1,131 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package errors
+
+import (
+	"encoding/json"
+	"regexp"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
```

Comment:
Does it make sense to put this file in a `errors/grpc` package? If not, we're need to vendor these big packages just to use our generic errors package when not using gRPC